helper/populate_db.py

```diff
@@ -54,21 +55,14 @@ def initialize_questions():
     return data
 
 
-def initialize_activities():
-    data = [
-        InterventionActivity(intervention_activity_id=1,
-                             intervention_activity_title="Relaxation exercise",
-                             intervention_activity_description="Go to this link to listen to the instructions for the relaxation exercise: ... .",
-                             intervention_activity_full_instructions='Now here are very detailed instructions for the relaxation exercise.'),
-        InterventionActivity(intervention_activity_id=2,
-                             intervention_activity_title="Reasons to quit",
-                             intervention_activity_description="Think back about the reasons why you want to quit smoking.",
-                             intervention_activity_full_instructions="These are very detailed instructions for reasons for quitting."),
-        InterventionActivity(intervention_activity_id=3,
-                             intervention_activity_title="Reasons to be physically active",
-                             intervention_activity_description="Think again about why you want to be physically active. Maybe you still have the list on your fridge.",
-                             intervention_activity_full_instructions="Now here are very detailed step-by-step instructions on reasons for PA.")
-    ]
+def initialize_activities(activities_file_path):
+    with open(activities_file_path, newline='') as csvfile:
+        csv_reader = csv.DictReader(csvfile)
+        data = [InterventionActivity(intervention_activity_id=int(row['activity_id']),
+                                     intervention_activity_title=row['activity_title'],
+                                     intervention_activity_description=row['activity_description'],
+                                     intervention_activity_full_instructions=row['activity_instructions'],
+                                     user_input_required=bool(int(row['input_needed']))) for row in csv_reader]
```

Comment:
I've seen nicer things than `bool(int())`... but it's not worth spending time on a helper function here

Reply:
I am also very unhappy with that :sweat_smile: